chain/chain/src/store.rs

```diff
@@ -2409,6 +2409,9 @@ impl<'a> ChainStoreUpdate<'a> {
                 store_update.delete(col, key);
                 self.chain_store.processed_block_heights.cache_remove(key);
             }
+            DBCol::ColCachedContractCode => {
```

Comment:
I don't think it makes sense to garbage collect this column. The management of this cache should be done separately from regular garbage collection code